tests/baselines/reference/types.forAwait.esnext.1.symbols

```diff
@@ -0,0 +1,55 @@
+=== tests/cases/conformance/types/forAwait/types.forAwait.esnext.1.ts ===

```

Comment:
Because they have `// @noEmit: true` at the top of the file. I am not checking emit with these tests, just type resolution. The .js output would be extraneous. There should be enough coverage in `emitter.forAwait.esnext.ts`, `emitter.forAwait.es2015.ts`, and `emitter.forAwait.es5.ts` for that.